### ISSUE TITLE
Improve link focus accessibility

### ISSUE THREAD

**Issue by meganem** [SU-SWS/open_framework#163]
1. Test focus in all themes.
2. Identify focus styles that are not visible, or not high enough contrast.
3. Fix those elements.

**Comment by meganem** [comment]
tested OFW and the only thing that looked bad was the form submit buttons, so fixed that.
